rpc/types_broadcast_transaction.go

```diff
@@ -40,14 +42,26 @@ type BroadcastInvokev0Txn struct {
 	InvokeTxnV0
 }
 
+func (tx BroadcastInvokev0Txn) GetCalldata() []*felt.Felt {
+	return tx.InvokeTxnV0.FunctionCall.Calldata
```

Comment:
NIT: you can also set this to `tx.Calldata` thanks to [Go Struct Embedding](https://gobyexample.com/struct-embedding)

Reply:
Done...